test.ts

```diff
@@ -579,3 +579,23 @@ Deno.test("dateTimeIsCorrect", function () {
   assertEquals(new Date(now + "Z"), new Date());
   db.close();
 });
+
+Deno.test("lastInsertedId", function () {
+  const db = new DB();
+
+  // By default, lastInsertRowId must be 0
+  assertEquals(db.lastInsertRowId, 0);
+
+  // Create table and insert value
+  db.query(`CREATE TABLE users (id INTEGER PRIMARY KEY, name VARCHAR(255))`);
+  db.query("INSERT INTO users (name) VALUES ('John Doe')");
+
+  // Now, the last inserted row id must be 1
+  assertEquals(db.lastInsertRowId, 1);
```

Comment:
I'd be happier with getting the correct value from an SQL statement, so to do something like this:

```js
const insertRowIds = [];
for (let i = 0; i < 10; i ++) {
  db.query("INSERT INTO users (name) VALUES ('John Doe')");
  insertRowIds.push(db.lastInsertRowId);
}

assertEquals(insertRowIds, [...db.query("SELECT rowid FROM users")].map(([i]) => i));
```

Reply:
Done!